docs/aqd.rst

```diff
@@ -0,0 +1,283 @@
+Interacting with smart contracts on the command line
+====================================================
+Solang Aqd (عَقد, meaning "contract" in Arabic), is a Command-Line Interface (CLI) tool 
+designed for easy interaction with smart contracts on Solana and Polkadot blockchains.
+It simplifies the process of deploying smart contracts and executing specific functions on these contracts. 
+This documentation provides essential guidance for using ``solang-aqd`` effectively.
+
+
+Installation
+____________
+
+To install `solang-aqd`, run this command:
+
+.. code-block:: bash
+
+    cargo install --force --locked aqd
+    
+To update to the latest version, use the same command.
+
+
+Submitting extrinsics to Polkadot on-chain
+__________________________________________
+
+The command line syntax for interacting with a program deployed on Polkadot is as follows:
+
+aqd polkadot [SUBCOMMAND] [OPTIONS]... [FILE]
+
+This means that the command line is ``aqd polkadot`` followed by a subcommand followed by any options described below,
+followed by the filename. The filename could be ``.wasm file``, ``.contract`` bundle, or ``.json`` metadata file.
+
+General Options (for all subcommands):
+++++++++++++++++++++++++++++++++++++++
+
+\-\-url *url*
+  The websockets URL for the substrate node. [default: ws://localhost:9944]
+
+\-\-network *network*
+  Specify the network name to use.
+  You can either specify a network name or a URL, but not both.
+
+  Network:
+
+  rococo
+    Contracts (Rococo) (Equivalent to ``--url wss://rococo-contracts-rpc.polkadot.io``)
+
+  phala-po-c5
+    Phala PoC-5 (Equivalent to ``--url wss://poc5.phala.network/ws``)
+
+  astar-shiden
+    Astar Shiden (Kusama) (Equivalent to ``--url wss://rpc.shiden.astar.network``)
+
+  astar-shibuya
+    Astar Shibuya (Tokio) (Equivalent to ``--url wss://rpc.shibuya.astar.network``)
+
+  astar
+    Astar (Equivalent to ``--url wss://rpc.astar.network``)
+
+  aleph-zero-testnet
+    Aleph Zero Testnet (Equivalent to ``--url wss://ws.test.azero.dev``)
+
+  aleph-zero
+    Aleph Zero (Equivalent to ``--url wss://ws.azero.dev``)
+  
+  t3rnt0rn
+    T3RN T0RN (Equivalent to ``--url wss://ws.t0rn.io``)
+  
+  pendulum-testnet
+    Pendulum Testnet (Equivalent to ``--url wss://rpc-foucoco.pendulumchain.tech``)
+
+-s, \-\-suri *suri*
+  Specifies the secret key URI used for deploying the contract (must be specified). For example:
```

Comment:
Maybe it'd be better to provide a link to more complete docs how this works

Reply:
Added a note to refer users to verbose documentation for more information